pagopa/pn-frontend-e2e
# PN-9308-e-2-e-destinatario-recapiti-cortesia-inserimento-mail

## Short description
There is a missing step in test related to the **PN-9308** as you can see in the [confluence page](https://pagopa.atlassian.net/wiki/spaces/PN/pages/868778282/Mittente+-+Mapping+test-scenari)

## List of changes proposed in this pull request
- added new feature step `And Nella pagina I Tuoi Recapiti si visualizza correttamente la sezione 'E-mail o numero di cellulare'`
- added related checks and methods


## How to test
Simply run `mvn clean test -Dcucumber.filter.tags=@TA_inserimentoEmailPF` and verify if the test works fine.
